libs/dh/admin/feature-user-roles-new/src/lib/details/tabs/audit-logs.component.html

```diff
@@ -19,20 +19,20 @@
   *transloco="let t; read: 'admin.userManagement.drawer.roles.tabs.history'"
 >
   @if (isLoading()) {
-    <div class="spinner">
+    <vater-flex justify="center">
       <watt-spinner />
-    </div>
+    </vater-flex>
   } @else {
-    @if (!hasFailed()) {
+    @if (!hasError()) {
       <watt-card-title>
-        <ng-container
-          *ngTemplateOutlet="
-            auditLogHeadline;
-            context: {
-              $implicit: dataSource.data.length,
-            }
-          "
-        />
+        <h4 *transloco="let hr; read: 'admin.userManagement.drawer.roles.tabs.history.changes'">
+          @let auditLogCount = dataSource.totalCount;
+          @if (auditLogCount === 1) {
+            {{ hr("singular", { auditLogCount }) }}
+          } @else {
+            {{ hr("plural", { auditLogCount }) }}
+          }
+        </h4>
```

Comment:
suggestion: wouldn't this work? No need for nested Transloco directive.

```suggestion
        <h4>
          @let auditLogCount = dataSource.totalCount;
          @if (auditLogCount === 1) {
            {{ t("changes.singular", { auditLogCount }) }}
          } @else {
            {{ t("changes.plural", { auditLogCount }) }}
          }
        </h4>
```